Font fun

mark-prins opened the pull request:
Fixes #492 - kinda. Possibly makes it worse

Spent a while looking at options, since I've had the fout show for me too! Found it hard to replicate actually, so decided after much playing around, that it isn't a huge problem.. and also that it might be better to have the text show than to delay display of the site. We already have react suspense, and if you are using chrome or firefox, there is a 3s FOIT (flash of invisible text) time anyway before the default font is used and then updated to the correct font.

I did add a fallback of `Arial` rather than the serif default, this is much closer to `Inter` so the fout is less obvious, maybe?

And then made things a little worse, possibly, by adding in the font weights that we are using - I did wonder why the fonts weren't looking exactly the same as in zeplin. turns out we weren't loading all the weights.

Then went offroad a little and fixed the table headers, which were looking like this:

### Stocktake
<img width="988" alt="Screen Shot 2021-12-21 at 11 08 10 AM" src="https://user-images.githubusercontent.com/9192912/146839985-edf3cb73-bdfc-4301-8765-2ccb46fe6d45.png">

### Outbound shipment
<img width="994" alt="Screen Shot 2021-12-21 at 11 07 25 AM" src="https://user-images.githubusercontent.com/9192912/146840054-cf2321c2-81d4-47c9-a99c-8454e98520fd.png">


and now look better 😀 
### Stocktake
<img width="982" alt="Screen Shot 2021-12-21 at 11 09 01 AM" src="https://user-images.githubusercontent.com/9192912/146840075-4f629e17-1432-4b65-9d21-4bb838af4f2d.png">


### Outbound shipment
<img width="996" alt="Screen Shot 2021-12-21 at 11 09 35 AM" src="https://user-images.githubusercontent.com/9192912/146840068-a1a4d0de-e616-4479-891b-b069f01e28e7.png">

joshxg commented on packages/host/public/index.html:
```diff
@@ -1,8 +1,10 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap"
```
Wouldn't these not work if you're on a lan?

We're using the `fontsource` package for `Inter` for self-hosting the font and we have all the different weights in node modules.. maybe they're just not getting loaded properly?

![image](https://user-images.githubusercontent.com/35858975/146845579-9cdedd3b-34cd-418a-9cd6-193bfc2c1e0e.png)

mark-prins replied:
huh. I didn't realise we were using `fontsource` oops. removing the `link` from the html does screw up the fonts though, so if there is fontsource it isn't used for the web version..??

joshxg reviewed (approved):
It's looking much better - I am a little confused about fonts if I'm honest.. I thought just adding the fontsource and importing worked like https://fontsource.org/docs/getting-started

But I suppose not 🤷

mark-prins commented:
ah! fountsource was only used for storybook, and the google fonts cdn was still used for the site.
I've changed the css references to use `fontsource` and also updated to use variable fonts ( soz IE 11 ). I'm told that variable fonts reduce the bundle size, so have swapped to those.
the change has now upped the bundle by 184kb .. but this is due to the hosting of the woff files, and it includes the various character sets which we don't use. and the files are browser-cacheable
